definitions/npm/redux-saga_v0.11.x/flow_>=v0.28.x/redux-saga_v0.11.x.js

```diff
@@ -0,0 +1,482 @@
+/* @flow */
+
+/* eslint-disable */
+
+/**
+ * Shared interfaces between the modules 'redux-saga' and
+ * 'redux-saga/effects'
+ */
+declare interface $npm$ReduxSaga$Channel {
```

Comment:
I think `put` method is missing (http://yelouafi.github.io/redux-saga/docs/api/index.html#channel)